project-base/app/tests/App/Functional/PersonalData/PersonalDataExportXmlTest.php

```diff
@@ -87,6 +87,9 @@ public function testExportXml(): void
         $watchdogProduct = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . '1');
         $watchdog = $this->createWatchdog($customerUser->getEmail(), $watchdogProduct);
 
+        $status->setCurrentLocale('en');
+        $complaint->getStatus()->setCurrentLocale('en');
```

Comment:
maybe we could render the expected file with Twig, so we're able to translate the strings, and in the end, assert the XML string with the string.
That way we may avoid setting an arbitrary locale that may not be available in the project.

Reply:
I have changed it, but I am not using twig as I would need loader for that